include/odp/api/spec/time.h

```diff
@@ -21,20 +21,24 @@ extern "C" {
 #include <odp/api/time_types.h>
 
 /** @defgroup odp_time ODP TIME
- *  Chip and CPU level wall clock time.
+ *  SoC global and CPU local wall clock time
+ *
  *  @{
  */
 
 /**
  * Current local time
  *
- * Returns current local time stamp value. The local time source provides high
- * resolution time, it is initialized to zero during ODP startup and will not
- * wrap around in at least 10 years.
- * Local time stamps are local to the calling thread and must not be shared
- * with other threads.
+ * Returns current CPU local time stamp value. The value can be used only by the calling thread,
+ * and only as long as it runs on the same CPU. Local time stamp values must not be shared with
+ * other threads.
+ *
+ * Local time stamp value advances with a constant rate defined by odp_time_local_res(). The rate
```

Comment:
Maybe it is too much promised that the time stamp value advances at a constant rate. Maybe it could be just said that the local time advances and is not affected by frequency scaling?